packages/frameworks-solid-start/src/utils.ts

```diff
@@ -0,0 +1,103 @@
+import type {
```

Comment:
I think maybe this could be something like `@auth/client`, as it's just using  Vanilla JS and is the same in SvelteKit Auth. Could re-export it so people would not need to install another package.

Note that in SvelteKit Auth, this solution is a workaround until we add something more "SvelteKit-like" that does not require client-side js.

Thinking of some form components.